robottelo/cli/base.py

```diff
@@ -243,7 +243,7 @@ def execute(
         )
         response = ssh.command(
             cmd.encode('utf-8'),
-            hostname=hostname or cls.hostname,
+            hostname=hostname or settings.server.hostname or cls.hostname,
```

Comment:
with satellite context manager, hostname for CLI tests was still pointing to default_sat, so all CLI functions were being executed on default_sat instead of module_destructive_sat, @jyejare Thanks for this fix 😊